### PR TITLE
Vm s63 footer

### PR DESCRIPTION
**Pull request by PizzaCat07**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,10 +10,15 @@
     "test": "jest"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.11",
```
Comment (doolaik) [inline package.json:13]: We aren't using mask view. Don't know why you installed it.

File: package.json [hunk 2]
```diff
@@ -10,10 +10,15 @@
...
     "@react-navigation/bottom-tabs": "^6.5.7",
+    "@react-navigation/drawer": "^6.6.2",
```
Comment (doolaik) [inline package.json:15]: Not using a drawer either

File: __tests__/__snapshots__/BottomTabNav.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (doolaik) [inline __tests__/__snapshots__/BottomTabNav.test.js.snap:1]: Your snapshot is obsolete

File: __tests__/BottomTabNav.test.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+describe('Testing react navigation', () => {
+  test('clicking on the bookmarks button takes you to the bookmarks screen', async () => {
+    const component = (
+      <NavigationContainer>
+        <AppNavigator />
+      </NavigationContainer>
+    );
+
+    render(component);
+    const homeScreen = await screen.queryByText('HomeScreen');
```
Comment (doolaik) [inline __tests__/BottomTabNav.test.js:16]: All these screen queries will fail once the placeholder screens are replaced; how should we handle that? Correct these tests once they are replaced?

Comment (PizzaCat07) [reply __tests__/BottomTabNav.test.js:16]: yes we will have to change some of the tests as the app is filled out, the tests will need to change as we start putting the components together.

**Comment by PizzaCat07** [comment]
snap shot removed.

**Review by doolaik — APPROVED** [review]
Fine

**Pull request merged**